src/python/pose_format/bin/pose_estimation.py

```diff
@@ -15,7 +15,7 @@ def load_video_frames(cap: cv2.VideoCapture):
     cap.release()
 
 
-def pose_video(input_path: str, output_path: str, format: str, additional_config: dict):
+def pose_video(input_path: str, output_path: str, format: str, additional_config: dict = {'model_complexity': 1}, progress: bool = True):
```

Comment:
I think `additional_config` should default to `None`. it knows how to handle it

Reply:
We had this line here, which would raise an error:

https://github.com/sign-language-processing/pose/blob/master/src/python/pose_format/bin/pose_estimation.py#L30

Do we still want by default `{'model_complexity': 1}`? I do not know the default behavior from Mediapipe and how much difference it brings.

Reply:
i see, in that case, keep it